<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMWindow</class>
 <widget class="QMainWindow" name="IMWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eIM-即时通信-Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="serverInfoContainer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>server info</string>
    </property>
    <widget class="QLabel" name="labelIp">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>51</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ip</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPort">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="port">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverIp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectBtn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRoom">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>room</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="roomNumber">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="connectionStatus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>571</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ready</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="msg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="infos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>611</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>serverIp</tabstop>
  <tabstop>port</tabstop>
  <tabstop>roomNumber</tabstop>
  <tabstop>connectBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
